cerbero/build/cookbook.py

```diff
@@ -55,25 +55,30 @@ class RecipeStatus (object):
     @ivar built_version: string with the last version built
     @type built_version: str
     @ivar file_hash: hash of the file with the recipe description
-    @type file_hash: int
+    @type file_hash: str
+    @ivar installed_files: installed files
+    @rtpe installed_files: list
     '''
 
     def __init__(self, filepath, steps=[], needs_build=True,
-                 mtime=time.time(), built_version='', file_hash=0):
+                 mtime=time.time(), built_version='', file_hash='',
+                 installed_files=[]):
         self.steps = steps
         self.needs_build = needs_build
         self.mtime = mtime
         self.filepath = filepath
         self.built_version = built_version
         self.file_hash = file_hash
+        self.installed_files = installed_files
 
     def touch(self):
         ''' Touches the recipe updating its modification time '''
         self.mtime = time.time()
 
     def __repr__(self):
-        return "steps: %r, needs_build: %r, mtime: %r, filepath: %r, built_version: %r, file_hash: %r" % \
-            (self.steps, self.needs_build, self.mtime, self.filepath, self.built_version, self.file_hash.hex())
+        return "steps: %r, needs_build: %r, mtime: %r, filepath: %r, built_version: %r, file_hash: %r\ninstalled_files: %r" % \
+            (self.steps, self.needs_build, self.mtime, self.filepath, self.built_version, self.file_hash,
```

Comment:
I don't know what it is changing but you removed the .hex() from self.file_hash

Reply:
Yeah, I changed the type of `file_hash` from int to str on purpose